components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -301,6 +302,32 @@ - (void)testCustomDialogPresentationElevation {
   XCTAssertEqual(shadowView.elevation, elevation);
 }
 
+- (void)testCustomShadowColor {
+  // Given
+  UIColor *fakeColor = UIColor.orangeColor;
+
+  // When
+  self.alert.shadowColor = fakeColor;
+
+  // Then
+  XCTAssertEqualObjects(self.alert.mdc_dialogPresentationController.trackingView.shadowColor,
```

Comment:
lets test the "copy" part too by making sure the pointers are not the same.

Reply:
This does tests to make sure that the copy is the same, since `[UIColor copy]` returns `self` they are the same pointer. `copy` in the header is to match `UIKit`please see [UIView documenation](https://developer.apple.com/documentation/uikit/uiview/1622591-backgroundcolor?language=objc). 

```objc
UIColor *fakeColor = UIColor.whiteColor;
XCTAssertNotEqual([fakeColor copy], fakeColor); // Fail, because they are the same pointer
XCTAssertEqualObjects([fakeColor copy], fakeColor); // Pass, because they are equal objects.
```

Reply:
We should not be testing that `copy` semantics do not return the same object. Returning the same immutable object is a very efficient implementation of `copy`.